server/controllers/index.js

```diff
@@ -1,7 +1,7 @@
 export * from "./dropQuestItem.js";
 export * from "./getLeaderboard.js";
 export * from "./handleGetDroppedAssetDataObject.js";
-export * from "./handleGetDroppedAssetsWithUniqueName.js";
+export * from "./handleGetQuestItems.js";
```

Comment:
Some filenames have handle in it, and others don't. Why is that?

Reply:
I hadn't gotten to these yet during the refactor and didn't want to do it as part of this because it'd make it harder to compare changes but I'll go ahead and make the change now that it's been reviewed